vpc/limits.go

```diff
@@ -151,6 +151,45 @@ var interfaceLimits = map[string]map[string]limits{
 			networkThroughput:        20000,
 		},
 	},
+	"c5": {
+		"large": limits{
+			interfaces:               3,
+			ipAddressesPerInterface:  10,
+			ip6AddressesPerInterface: 10,
+			// Maybe?
```

Comment:
add `TODO` to confirm

Reply:
@aather Can you confirm the network throughput numbers here? Assuming best case?